types/braintree-web-drop-in/braintree-web-drop-in-tests.ts

```diff
@@ -0,0 +1,105 @@
+import * as dropin from "braintree-web-drop-in";
```

Comment:
There is no package with this name. Also looking at `https://github.com/braintree/braintree-web-drop-in#full-example` doesn't seem like a module. You want to fix the tests and definitions to have global file declarations instead. 
Please refer to https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html for more information.

Reply:
The package is here: https://www.npmjs.com/package/braintree-web-drop-in

Looking through TypeScript's handbook, a global package has "one or more assignments to `window.someName`", which `braintree-web-drop-in` does not. 

If it's any consolation, I've been using my modular type definitions with `braintree-web-drop-in` for almost all of its documented methods/properties, and I haven't run into any issues.

Reply:
You still need to expose all this into Global since that's what the usage shows. Also add test to verify global usage as well. 
Refer [this](https://github.com/DefinitelyTyped/DefinitelyTyped#how-do-i-write-definitions-for-packages-that-can-be-used-globally-and-as-a-module) for more details.

Reply:
Thank you for the help 👌